controllers/users.controller.js

```diff
@@ -47,6 +47,30 @@ function Login (req, res){
         });
 
 }
+/* 
+    Action : complete the `UpdateProfile` function, checked to be sure that the requested loan is not `null`
+    Date : 1st July 2018
+*/
+function UpdateProfile (req, res){
+    const userId = req.params.id;
+
+    LoansModel.find({ "userId": userId}, function (err, docs) {
```

Comment:
Why do you need to verify loan before update profile ?

Is there any valid reason for this ?